### PR TITLE
Add option to connect to voice gateways manually

### PR DESCRIPTION
**Pull request by BrandtHill**

### CODE REVIEW

File: docs/static/Intro.md [hunk 1]
```diff
@@ -80,7 +80,8 @@ The following fields are also supported:
  - `ffmpeg` - Specifies the path to the `ffmpeg` executable for playing audio. Defaults to `"ffmpeg"`.
  - `youtubedl` - Specifies the path to the `youtube-dl` executable for playing audio with youtube-dl support. Defaults to `"youtube-dl"`.
  - `gateway_intents` - This field takes a list of atoms representing gateway intents for Nostrum to subscribe to from the Discord API. More information can be found in the [gateway intents](gateway-intents.html) documentation page.
- - `audio_timeout` - Milliseconds that input must begin generating audio by upon invoking `play`. More information about this option can be found in the [voice](voice.html) documentation page. Defaults to `20_000` (20s). 
+ - `audio_timeout` - Milliseconds that input must begin generating audio by upon invoking `play`. More information about this option can be found in the [voice](voice.html) documentation page. Defaults to `20_000` (20s).
+ - `voice_auto_connect` - This will determine if Nostrum automatically connects to voice websockets gateways upon joining voice channels. Defaults to `true`.
```
Comment (jchristgit) [inline docs/static/Intro.md:84]: What do you think about adding a hint to `connect_to_gateway` here?

**Comment by TeeSeal** [comment]
Tested these changes. Works as expected 🎉

File: docs/static/Intro.md (hunk 1 above)
Comment (BrandtHill) [reply docs/static/Intro.md]: Added detail

**Review by jchristgit — APPROVED** [review]
Thanks, LGTM!

**Pull request merged**